modules/homefinder/widgets/class-affiliate-search-widget.php

```diff
@@ -0,0 +1,208 @@
+<?php
+/**
+ * HomeFinder Affiliate Search Widget (http://www.homefinder.com/widgets/)
+ *
+ * @package RE-PRO
+ */
+
+	/* Exit if accessed directly. */
+if ( ! defined( 'ABSPATH' ) ) { exit; }
+
+/**
+ * HomeFinderAffiliateSearch class.
+ *
+ * @extends WP_Widget
+ */
+class HomeFinderAffiliateSearch extends WP_Widget {
```

Comment:
__[code_sniffer]__
* Each class must be in a namespace of at least one level (a top-level vendor name)
* Opening brace of a class must be on the line after the definition

<!-- Comment by [SideCI](https://sideci.com) -->